identify/extensions.py

```diff
@@ -78,6 +80,7 @@
     'hcl': {'text', 'hcl'},
     'hh': {'text', 'header', 'c++'},
     'hpp': {'text', 'header', 'c++'},
+    'hrl': {'text', 'header', 'erlang'},
```

Comment:
while the current `header` tag isn't great -- I think it kinda means C-like headers so this would break tools expecting those?

Reply:
I'm not 100% sure what you mean. Erlang header files usually contain macros shared between source files. Do you have an example on how it's used in c?

Reply:
it's that it's already used for c likes so reusing the tag for erlang would break existing things

Reply:
hm, ok. I've removed 'header' for now, let's see how it works and in worst case I'll add a 'header_erlang' tag or something then.